jobtastic/task.py

```diff
@@ -15,8 +15,11 @@
 
 import psutil
 
-from celery.datastructures import ExceptionInfo
-from celery.five import class_property
+from billiard.einfo import ExceptionInfo
+try:  # Celery 3
+    from celery.five import class_property
+except ImportError:  # Celery 4
+    from celery.local import class_property
```

Comment:
i feel like celery4 should be the default and celery3 the exception.
but not a big deal